presto-docs/src/main/sphinx/clients/superset.rst

```diff
@@ -0,0 +1,37 @@
+===============
+Apache Superset
+===============
+
+`Apache Superset <https://superset.apache.org/>`_ is an open source data exploration tool that is part of the Apache Project. 
```

Comment:
delete "that is part of the Apache Project." Apache is so large these days that barely means anything.